01/components/Table.js

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import TableHeader from "./TableHeader";
+import TableBody from "./TableBody";
+import TableRow from "./TableRow";
+import TableFooter from "./TableFooter";
+
+class Table extends React.Component {
+  tableStyles = {
+    marginInline: "auto",
+    marginTop: "20px",
+    border: "1px solid black",
+  };
```

Comment:
Jeśli używasz plików CSS to już bym przy nich został, a nie osadzał jeszcze style "inline"